src/js/components/RangeInput/__tests__/RangeInput-test.js

```diff
@@ -105,4 +105,36 @@ describe('RangeInput', () => {
     expect(container.firstChild).toMatchSnapshot();
     expect(onChange).toBeCalledTimes(1);
   });
+
+  test('inputValue', () => {
```

Comment:
Could we add a test with one of the keys to open?
https://github.com/grommet/grommet/pull/4986/files#diff-b99e2906891e71a7ce78554badfd2fd18f035cd8bc4caff6c595d0cd64cfd4aaR7

Reply:
Nice!!! Thanks a lot